DashKitUI/PumpManager/DashPumpManager+UI.swift

```diff
@@ -66,11 +66,12 @@ extension DashPumpManager {
 extension DashPumpManager {
 
     public func syncScheduleValues(for viewController: BasalScheduleTableViewController, completion: @escaping (SyncBasalScheduleResult<Double>) -> Void) {
-        setBasalSchedule(dailyItems: viewController.scheduleItems) { (error) in
-            if let error = error {
+        syncBasalRateSchedule(items: viewController.scheduleItems) { result in
+            switch result {
+            case .success(let schedule):
+                completion(.success(scheduleItems: schedule.items, timeZone: schedule.timeZone))
+            case .failure(let error):
```

Comment:
[nit] Do we have any team coding standards about order of `.failure` and `.success` in `switch`? I know it doesn't matter at all, but I've always done failure first (just for consistency). It isn't a big deal so don't bother to change, just curious. 

cc: @ps2

Reply:
I default to `success` first but don't have a strong opinion.